content/motivation.md

```diff
@@ -46,7 +46,10 @@ to establish accuracy:
        expected_result = 37.777777
        assert abs(temp_c - expected_result) < 1.0e-6
    ```
+   ```{code-tab} c/c++
```

Comment:
You need `c++` as argument, otherwise the Pygments lexer gets confused on how to generate highlighting

Reply:
Hey, I have updated the code and added cpp code also. please review and merge